src/doc/yaml/enrollment.yaml

```diff
@@ -0,0 +1,652 @@
+              format: byte
+      responses:
+        '201':
+          description: Operation successful
+          content:
+            application/json:
+              schema:
+                type: object
+                required:
+                  - bufferId
+                properties:
+                  enrollmentId:
+                    type: string
+        '400':
+          description: Bad request
+        '403':
+          description: Update not allowed
+        '404':
+          description: Unknown record
+        '500':
+          description: Unexpected error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+            
+  /v1/enrollment/{enrollmentId}/buffer/{bufferId}:
+    get:
+      tags:
+        - Enrollment
+      summary: This service is used to get the buffer of the images
+      operationId: getBuffer
+      parameters:
+        - name: enrollmentId
+          in: path
+          description: the id of the enrollment
+          required: true
+          schema:
+            type: string
+        - name: bufferId
+          in: path
+          description: the id of the buffer
+          required: true
+          schema:
+            type: string
+        - name: transactionId
+          in: query
+          description: The id of the transaction
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: Read successful
+          content:
+            plain/text:
+              schema:
+                description: The image
+                type: string
+                format: byte
+        '400':
+          description: Bad request
+        '403':
+          description: Update not allowed
+        '404':
+          description: Unknown record
+        '500':
+          description: Unexpected error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+
+components:
+  schemas:
+    Error:
+      type: object
+      required:
+        - code
+        - message
+      properties:
+        code:
+          description: Error code
+          type: integer
+          format: int32
+        message:
+          description: Error message
+          type: string
+    Enrollment:
+      type: object
+      required:
+        - status
+        - enrollmentType
+        - biometricData
+      properties:
+        encounterId:
+          type: string
+        status:
+          type: string
+          enum: [ACTIVE, INACTIVE]
+        enrollmentType:
+          type: string
+          description: Type of the enrollment
+        enrollmentFlags:
+          type: array
+          items:
+            type: string
+          minItems: 1
+          uniqueItems: true
+        requestData:
+          type: array
+          items:
+            type: string
+          minItems: 1
+          uniqueItems: true
```

Comment:
Should be a dictionary and not a list to be aligned on the functional description of chapter 5.4.4

Reply:
RequestData can be a disctionary but enrollmentFlags should be a list? Or should I make it a dictionary?

Reply:
Will fix this